_posts/2023-06-17-call-for-venue-proposal-2024.md

```diff
@@ -29,32 +29,23 @@ We need some information about your city, about the venue, the hotel, and travel
 	* Location
 	* Transportation options (how do people get there from the airport?)
 	* Attendance capacity
-	* Layout of function space:
+	* Assurance that the venue can provide the following spaces (A floorplan with explanation of what will go where would be great.):
 		* Keynote room (also Sessions Track A) to accommodate up to 450 people, classroom style (tables with chairs)
 		* Room for Sessions Track B to accommodate up to 200 people, classroom style (tables with chairs)
 		* Space for sponsors to set up exhibit booths/tables; at least 12, and up to 16, sponsors with either 6’ tables or 10’ x 10’ booth spaces. If more sponsor booth space is available, that's a plus.
 		* Green room for speakers to prepare (generally a room that will seat 25 for a workshop is the right size)
-		* Location of registration tables
-		* Location of meals and breaks with sufficient seating and tables. If serving and/or seating areas cannot accommodate 450 people at once, indicate the room capacity and timing to move 450 people through.
-		* Location of supplemental beverage stations for daylong service.
+		* Space for registration tables
+		* Room for meals and breaks with sufficient seating and tables. If serving and/or seating areas cannot accommodate 450 people at once, indicate the room capacity and time required to move 450 people through.
+		* Space for supplemental beverage stations for daylong service.
 		* Secure overnight storage for vendor booth supplies and conference registration desk items.
 
-	* Catering:
-		* Tutorials day—100 people: Breakfast, morning break, lunch, afternoon break, and all day beverage service. (All day beverage service includes water stations, coffee, assorted black and herbal teas, and a variety of soft drinks.)
-		* Main conference, 3 days—450 people per day: Breakfast, morning break, lunch, afternoon break, and all day beverage service. (All day beverage service includes water stations, coffee, assorted black and herbal teas, and a variety of soft drinks.)
-		* Sprints, 2 days—75 people each day: Breakfast, morning break, lunch, afternoon break, and all day beverage service. (All day beverage service includes water stations, coffee, assorted black and herbal teas, and a variety of soft drinks.)
+	* Explanation of catering options available at the venue that can cover:
+		* 450 people per day: Breakfast, morning break, lunch, afternoon break, and all day beverage service.
 		* Catering must have options for celiacs, vegetarians, and vegans of equal quality and variety as main menu. Estimate 15% vegetarian, 5% vegan, and 5-10% gluten-free (some of which may overlap with the other two groups).
-
-	* Costs for:
-		* Function space
-		* Catering
-		* Audio-visual services
```

Comment:
@adamfast should we pull this or reword it?

Reply:
Maybe suggest a local A/V vendor or two if they know of one?